packages/daimo-api/src/cron.ts

```diff
@@ -68,12 +69,31 @@ export class Crontab {
 
     if (depositEth < 0.001) {
       this.telemetry.recordClippy(
-        `Paymaster deposit too low: ${depositEth} eth`,
+        `Paymaster ${daimoPaymasterAddress} deposit too low: ${depositEth} eth`,
         "error"
       );
     } else if (depositEth < 0.01) {
       this.telemetry.recordClippy(
-        `Paymaster deposit low: ${depositEth} eth`,
+        `Paymaster ${daimoPaymasterAddress} deposit low: ${depositEth} eth`,
+        "warn"
+      );
+    }
+  }
+
+  async checkFaucetBalance() {
+    const faucetAddr = this.vc.walletClient.account.address;
+    const balance = await this.vc.publicClient.getBalance({
+      address: faucetAddr,
+    });
+    const balanceEth = Number(formatEther(balance));
+    if (balanceEth < 0.002) {
+      this.telemetry.recordClippy(
+        `Faucet ${faucetAddr} balance too low: ${balanceEth} eth`,
+        "error"
+      );
+    } else if (balanceEth < 0.02) {
+      this.telemetry.recordClippy(
+        `Faucet ${faucetAddr} balance low: ${balanceEth} eth`,
```

Comment:
🚀 

we can even make this 0.01 (too low) and 0.05 (low) so we have time to fix it

Reply:
0.02 is nearly 50+ user onboardings at current gas costs, should be fine